homeassistant/components/shelly/utils.py

```diff
@@ -221,12 +223,27 @@ def get_shbtn_input_triggers() -> list[tuple[str, str]]:
 async def get_coap_context(hass: HomeAssistant) -> COAP:
     """Get CoAP context to be used in all Shelly Gen1 devices."""
     context = COAP()
+
+    adapters = await network.async_get_adapters(hass)
+    LOGGER.debug("Network adapters: %s", adapters)
```

Comment:
Discussed on discord that it would be helpful for this information to appear in the shelly debug logs when enabled from the UI

Reply:
I added it only because users usually enable debug from the ui and this will be only for `shelly` component and `aioshelly` so to get the info easily, IMHO the debug on the shelly side is useful.